views/loginView.ejs

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Logowanie</title>
+    <link rel='stylesheet' href='/stylesheets/login.css' />
+  </head>
+  <body>
+    <div class="container">
+        <div class="header">
+            <h2>Member login</h2>
+        </div>
+        <div class="main">
+            <div class="linput"> <input type="email" placeholder="email"></div>
+            <div class="linput"> <input type="email"placeholder="password"></div>
+            <button class="btn">LOGIN</button>
+            <div class="forgot"><a href="#">Forgot password?</a></div>
+            <div class="newAccount"><a href="#">Create new Account</a></div>
+        </div>
```

Comment:
wrap this in form field like this: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/forms

Reply:
done